score.go

```diff
@@ -202,13 +202,9 @@ func (ps *peerScore) SetTopicScoreParams(topic string, p *TopicScoreParams) erro
 	ps.Lock()
 	defer ps.Unlock()
 
-	old, exist := ps.params.Topics[topic]
+	old, _ := ps.params.Topics[topic]
```

Comment:
you can just remove the underscore completely.

Reply:
Done